types/audit-details.d.ts

```diff
@@ -167,6 +167,15 @@ declare global {
         nodeLabel?: string;
       }
 
+      export interface SourceLocationValue {
+        type: 'source-location';
+        /** False when url came from a magic comment sourceURL. */
+        urlIsNetworkResource: boolean;
```

Comment:
this variable name looks weird, I have regrets... 😆 

`isNetworkResourceUrl`?

Reply:
I'll just bring this to the surface again :)

FYI @brendankenny since you've specifically called out wanting to be careful of new details type, this is my biggest hesitation

Reply:
i dont think this variable is named consistently with how it is used.  it seems to be the determining factor in whether its rendered as a textURL (actual `<a href>`) vs text (plain `div`)? 

but then in devtools both cases will be linked to the Sources panel, right?

Reply:
> i dont think this variable is named consistently with how it is used. it seems to be the determining factor in whether its rendered as a textURL (actual <a href>) vs text (plain div)?

Yes, and with the additional caveat that in DevTools, it gets completely blown out, but only if the linkifying was successful for the data given.

> but then in devtools both cases will be linked to the Sources panel, right?

correct. `{url, line, column}` is enough info for devtools to link to the sources panel, regardless if the url is from the sourceURL or the original url.

I think replacing this with `urlProvider: 'comment' | 'network'` might address all your concerns here.

Reply:
`urlProvider: 'comment' | 'network'` sounds great to me!